### PR TITLE
Update delegates.json

### PR DESCRIPTION
**Pull request by TAOVali**
"5EhvL1FVkQPpMjZX4MAADcW42i3xPSF1KiCpuaxTYVr28sux": {
        "name": "TAO-Validator.com", 
        "url": "www.tao-validator.com", 
        "description": "TAO-Validator.com is a highly secure validator on the Bittensor Network. Maximize your return when staking with TAO-Validator.com"

### CODE REVIEW

**Comment by camfairchild** [comment]
Hi, you're still missing the signature. Without this we can't verify your hotkey is the one that made this commit. You can also use the polkadotjs explorer Sign&Verify if that's easier for you.

**Comment by TAOVali** [comment]
Verified True
        Validator information: {"5EhvL1FVkQPpMjZX4MAADcW42i3xPSF1KiCpuaxTYVr28sux": {"name": "TAO-Validator.com", "url": "www.tao-validator.com", "description": "Maximize your return when staking with TAO-Validator.com. TAO-Validator.com is a highly secure validator that aims to become one of the top contributing entities to Bittensor."}}
        Validator signature: 4036991069d7f3a43dff2ba2592fbe5af820eb6ff96d1fb78f1bcd8d310ba8751e25ea14397e075368a9a0f1b1b176166c56351db36f2d3868ac61c2571a1981

**Comment by TAOVali** [comment]
Ah apologies, I did not understand that step 6 but all good now. I’ve updated it (verified with my hotkey in the PR), could you please let me know if we’re good now?
Thanks!

File: delegates.json [hunk 1]
```diff
@@ -63,5 +63,10 @@
         "name": "Tensor.Exchange", 
         "url": "www.tensor.exchange", 
         "description": "Bittensor's first community OTC exchange"
-    }
+    },
+    "5EhvL1FVkQPpMjZX4MAADcW42i3xPSF1KiCpuaxTYVr28sux": {
+        "name": "TAO-Validator.com", 
+        "url": "www.tao-validator.com", 
+        "description": " Maximize your return when staking with TAO-Validator.com. TAO-Validator.com is a highly secure validator that aims to become one of the top contributing entities to Bittensor."
```
Comment (camfairchild) [inline delegates.json]:
```suggestion
        "description": "Maximize your return when staking with TAO-Validator.com. TAO-Validator.com is a highly secure validator that aims to become one of the top contributing entities to Bittensor."
```
Didn't matcg sig

**Review by camfairchild — APPROVED** [review]
Passes validation, ty

**Comment by TAOVali** [comment]
Hi Cameron, I just looked up btcli list_delegates but I still don’t see the description with our vali?
Does it take some extra time or should it be instant and something went wrong?

Thank you for looking at it.

**Comment by TAOVali** [comment]
Guys, it’s still not up in CLI. This is getting very annoying, please troubleshoot asap. Thank you

From: ***@***.***
Sent: Thursday, March 23, 2023 10:04 PM
To: opentensor/bittensor ***@***.***>; opentensor/bittensor ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [opentensor/bittensor] Update delegates.json (PR #1217)

Hi Cameron, I just looked up btcli list_delegates but I still don’t see the description with our vali?
Does it take some extra time or should it be instant and something went wrong?

Thank you for looking at it.

**Comment by camfairchild** [comment]
> it’s still not up in CLI. This is getting very annoying  

Hi, it's already in the CLI as of 16 hours ago. Please let me know if you need any help troubleshooting on your end.
Screenshot 2023-03-24 at 9 24 36 AM

**Comment by TAOVali** [comment]
Dear Cameron,

Regarding PR 1217 and PR 1229, I just saw you closed it with comment “Closed as already completed”
However, our details are not visible in the CLI, see visual below, the purple line is us. Please let me know how we are going to get our details on there, thank you

***@***.***


From: ***@***.***
Sent: Friday, March 24, 2023 9:22 AM
To: 'opentensor/bittensor' ***@***.***>; 'opentensor/bittensor' ***@***.***>
Cc: 'Author' ***@***.***>
Subject: RE: [opentensor/bittensor] Update delegates.json (PR #1217)

Guys, it’s still not up in CLI. This is getting very annoying, please troubleshoot asap. Thank you

From: ***@***.******@***.***>
Sent: Thursday, March 23, 2023 10:04 PM
To: opentensor/bittensor ***@***.******@***.***>>; opentensor/bittensor ***@***.******@***.***>>
Cc: Author ***@***.******@***.***>>
Subject: RE: [opentensor/bittensor] Update delegates.json (PR #1217)

Hi Cameron, I just looked up btcli list_delegates but I still don’t see the description with our vali?
Does it take some extra time or should it be instant and something went wrong?

Thank you for looking at it.

**Comment by camfairchild** [comment]
Please see my above comment with a screenshot showing the information present in the master branch.

If there are any details you would like changed from the current display, feel free to make a new PR with these changes.

**Pull request merged**